### PR TITLE
feat: added card component

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Made cards of different types like horizontal, vertical and overlay over cards 
Look into https://vibe-ui-4szukszd8-bhargavirdy.vercel.app/docs/cards.html for changes
image

### CODE REVIEW

File: components/badge.css [hunk 1]
```diff
@@ -4,15 +4,17 @@
 .badge{
     position: absolute;
     border-radius: 50%;
-    border: 2px solid var(--white);
 }
 .badge-red{
+    border: 2px solid var(--white);
```
Comment (dev-enforced) [inline components/badge.css:9]: You can store this value 2px solid var(--white) in your variables file as a badge-border variable for easy access.

File: docs/cards.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Buttons | VibeUI</title>
```
Comment (dev-enforced) [inline docs/cards.html:8]: Change this to Cards.

**Review by dev-enforced — APPROVED** [review]
Code looks good to me !!The cards also look good and responsive !!
Attached some changes below:

File: docs/cards.html (hunk 2 above)
Comment (bhargaviRdy) [reply docs/cards.html:8]: oops! Will change

File: components/badge.css (hunk 1 above)
Comment (bhargaviRdy) [reply components/badge.css:9]: thanks for the suggestion, Will do the change

**Review by harshitadaswani — APPROVED** [review]
The code structure looks good! You can merge it

**Pull request merged**